maestrowf/maestro.py

```diff
@@ -334,6 +336,11 @@ def setup_argparser():
     run.add_argument("--dry", action="store_true", default=False,
                      help="Generate the directory structure and scripts for a "
                      "study but do not launch it. [Default: %(default)s]")
+    run.add_argument("--draw", type=str, action="append", default=[],
```

Comment:
Are there any thoughts on enabling this feature to be used without having to submit a Maestro run?

Reply:
that already works

Reply:
ok, i need to clarify: it works with dry run, so it doesn't run anything, but it does do all the workspace creation and currently saves the dag_vis outputs into that workspace.  will look into a -no-workspace option

Reply:
Where would the `--no-workspace` option be added. It wouldn't make sense to have that be standalone. It'd have to be explicitly paired with `--dry` and/or `--draw`?

Reply:
So, that's a good question.  While that thought/idea hadn't been really fleshed out at the time, I think it'd have to be tied to --draw. Does it really make any sense to have it tied to --dry? That would seem to defeat the purpose of dry runs.

Reply:
Oh -- good call. I went too far with the logic on that one. I think there is a way to pair CLI paramters, but I'm not sure how that displays in help.

Reply:
Another suggestion for the command line option in particular -- look at the `choices=[]` optional field for `add_argument`. That lets you enforce specific choices for command line flags.